nikola/plugins/command/status.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+
+# Copyright © 2012-2015 Roberto Alsina and others.
+
+# Permission is hereby granted, free of charge, to any
+# person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the
+# Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish,
+# distribute, sublicense, and/or sell copies of the
+# Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice
+# shall be included in all copies or substantial portions of
+# the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY
+# KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE
+# WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR
+# PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS
+# OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR
+# OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+# SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+from __future__ import print_function
+import io
+import os
+from datetime import datetime
+from dateutil.tz import gettz, tzlocal
+
+from nikola.plugin_categories import Command
+
+
+class CommandDeploy(Command):
+    """ Site status. """
+    name = "status"
+
+    doc_usage = "[[preset [preset...]]"
+    doc_purpose = "display site status"
+    doc_description = "Show information about the posts and site deployment."
+    logger = None
+
+    def _execute(self, command, args):
+
+        self.site.scan_posts()
+
+        timestamp_path = os.path.join(self.site.config["CACHE_FOLDER"], "lastdeploy")
+
+        try:
+            with io.open(timestamp_path, "r", encoding="utf8") as inf:
+                last_deploy = datetime.strptime(inf.read().strip(), "%Y-%m-%dT%H:%M:%S.%f")
+                last_deploy_offset = datetime.utcnow() - last_deploy
+        except (IOError, Exception):
+            print("It does not seem like you’ve ever deployed the site (or cache missing).")
+
+        if last_deploy:
+
+            if last_deploy_offset.days > 0:
+                last_deploy_offsetstr = "{0} days and {1} hours".format(str(int(last_deploy_offset.days)), str(int(last_deploy_offset.seconds / 60 / 60)))
```

Comment:
Why do you need this `str(int())` dance?

Just use `{0:.0f} days and {1:.0f} hours` as the format string.  It will guarantee zero decimal places.  Even if not for this, str.format() does not need strings as its arguments — and it’s even better to give ints and floats so you can format them more precisely.